docs/docs/benchmark.md

```diff
@@ -94,3 +124,43 @@ This is done by:
 <li><a href="new/study">Creating a new study</a> with the name of the benchmark as the alias.</li>
 <li>Adding new tasks to the study by <i>tagging</i> them with the name of the benchmark.</li>
 </ul>
+
+Detailed steps:
+- Create a new study with name and description
+- Set flag to "private" - we dont want people adding tasks here
+- Set "suite" flag
+- Find the study tag study_X
+- Tag all tasks with study_X
+- You can add other tags as well. For each you can set:
+    - User IDs (who can extend study with that tag)
+    - Beginning/end dates (only stuff tagged within those dates are part of the study)
+- You do not have to tag any runs with study_X, these will be linked automatically
```

Comment:
Is this really true?

Reply:
I wasn't sure, but this was in the notes from the Paris meeting. Can you explain the process better? How do you add runs to a study?

Reply:
Yes, but the biggest problem with Paris minutes is that a lot of things got discussed, new ideas were introduced, without clear consensus and more importantly without who is going to implement it (but that last question is usually answered implicitly)

Reply:
Indeed :(
For clarity, can you list what is currently not possible yet? Then we need to see who has the time (if anyone), and maybe open issue for these.

Reply:
We need a way to move forward here. Let's break it down so we can discuss this.
```
- You can add other tags as well. For each you can set:
    - User IDs (who can extend study with that tag)
    - Beginning/end dates (only stuff tagged within those dates are part of the study)
```
Looking at the API docs, this seems to not be supported in the API. The dataset does keep this info. Are we missing the API calls for this? 
```
- You do not have to tag any runs with study_X, these will be linked automatically
```
This is what I remember @janvanrijn saying, but maybe I misremembered. Can you clarify?

The underlying question is: how can we create a new suite and make sure other people can't mess with it. Anything that does that is fine. If we create a special API call for this, that is fine by me, too. At this time it would be nice to have a concrete, workable proposal that will work in the long term and is easy to use and maintainable. If the above design is not OK, then let's find a better one.